src/test/kotlin/org/rust/lang/core/resolve/RsPackageLibraryResolveTest.kt

```diff
@@ -557,7 +557,28 @@ class RsPackageLibraryResolveTest : RsResolveTestBase() {
     """)
 
     @MockEdition(CargoWorkspace.Edition.EDITION_2018)
-    fun `test resolve bang proc macro with use item`() = stubOnlyResolve("""
+    fun `test resolve bang proc definition in macro crate itself`() = stubOnlyResolve("""
+    //- dep-proc-macro/lib.rs
+        #[proc_macro]
+        pub fn example_proc_macro_1(item: TokenStream) -> TokenStream { item }
+
+        #[proc_macro]
+        pub fn example_proc_macro_2(item: TokenStream) -> TokenStream { example_proc_macro_1(item) }
+                                                                        //^ dep-proc-macro/lib.rs
```

Comment:
Let's add a test with mod inside proc macro crate:
```rust
mod foo { pub fn bar() {} }
use foo::bar;
       //^
```
(looks like this test fails with your solution)

Reply:
Done